package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "wptdashboard",
   "devDependencies": {
-    "standard": "*",
-    "eslint-plugin-html": "*"
+    "eslint-plugin-html": "*",
+    "standard": "^10.0.3"
```

Comment:
Why more specific?

Reply:
Not sure what auto-updated this. Reverted.